### PR TITLE
create IaaS Breakout Session for the DNS Standard

### PR DESCRIPTION
**Pull request by artificial-intelligence**
Signed-off-by: Sven Kieske <kieske@osism.tech>

### CODE REVIEW

**Comment by artificial-intelligence** [comment]
We should discuss if the timing fits for everyone.

I'm also not 100% sure who to fill in as "coordinator" @fkr please complain if you are not the right person for this.

**Review by fkr — APPROVED** [review]
thanks!

File: teams/iaas/onetimes.yml [hunk 1]
```diff
@@ -53,3 +53,16 @@ events:
       Dial-In: +49-221-292772-611
       Coordinator: Patrick Thiem <patrick.thiem[at]cloudandheat.com>
     location: "https://conf.scs.koeln:8443/SCS-Tech"
+- summary: "IaaS Breakout room - DNS Standardization"
+    begin: 2024-05-10 14:05:00
```
Comment (frosty-geek) [inline teams/iaas/onetimes.yml:57]: is this really ment to happen in may?

Comment (artificial-intelligence) [reply teams/iaas/onetimes.yml]: thanks for catching that! of course not, this was a c&p mistake :grin:

**Comment by artificial-intelligence** [comment]
I changed the date now to 2024-06-17, which nobody objected to in the iaas call happening right now.

**Review by markus-hentsch — APPROVED** [review]

**Comment by garloff** [comment]
Needed to force this, as DCO checks still seem broken on github.

**Pull request merged**